### PR TITLE
New listitem 2

### PR DESCRIPTION
**Pull request by mardy**
This updates the Background tab.

### CODE REVIEW

File: src/SystemSettings/ListItems/ThinDivider.qml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import Ubuntu.Components.ListItems 1.3 as ListItem
+
+ListItem.ThinDivider {
```
Comment (cibersheep) [inline src/SystemSettings/ListItems/ThinDivider.qml:21]: Wouldn't you define this as:

```
import QtQuick 2.9
import Ubuntu.Components 1.3

Rectangle {
    width: parent.width
    height: units.dp(1)
    color: theme.palette.normal.base
}
```

or are you planning to change this ListItems dependency later on?

Comment (mardy) [reply src/SystemSettings/ListItems/ThinDivider.qml:21]: Yes, I plan to get rid of this later on; for the time being I'm just trying to reduce the usage of obsolete items to the `src/SystemSettings/ListItems` module.

What would you say about "undeprecating" the ThinDivider in the UITK (and maybe implementing it as you are suggesting here)?

Comment (cibersheep) [reply src/SystemSettings/ListItems/ThinDivider.qml:21]: Sounds like a plan :)

Oh, you mean «undeprecating» the ThinDivider but not the ListItems? Sounds good as we use it all over the system

Comment (mardy) [reply src/SystemSettings/ListItems/ThinDivider.qml:21]: Yes! :-)

File: po/ubuntu-system-settings.pot [hunk 2]
```diff
@@ -3048,833 +3062,854 @@ msgstr ""
...
+#. TRANSLATORS: This is a keyword or name for the hotspot plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the vpn plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the flight-mode plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the wifi plugin which is used while searching
+#. TRANSLATORS: This is a keyword or name for the cellular plugin which is used while searching
+#: ../obj-arm-linux-gnueabihf/po/settings.js:88
+#: ../obj-arm-linux-gnueabihf/po/settings.js:112
+#: ../obj-arm-linux-gnueabihf/po/settings.js:138
+#: ../obj-arm-linux-gnueabihf/po/settings.js:276
+#: ../obj-arm-linux-gnueabihf/po/settings.js:368
```
Comment (UniversalSuperBox) [inline po/ubuntu-system-settings.pot]: Any idea why it's putting files from the build directory into the translations file? It appears that it's happened before, do we fix it now?

**Review by UniversalSuperBox — DISMISSED** [review]
Approved for merge, but please see comment before doing so.

File: po/ubuntu-system-settings.pot (hunk 2 above)
Comment (mardy) [reply po/ubuntu-system-settings.pot]: Indeed I submitted the pot file by mistake. I've removed that from this MR, and I've created https://github.com/ubports/system-settings/pull/215 which should also address your remark.

**Review by UniversalSuperBox — APPROVED** [review]

**Pull request merged**